src/minicraft/screen/TitleDisplay.java

```diff
@@ -242,13 +224,12 @@ public void render(Screen screen) {
 		"Milk is an idea!",
 		"Creeper, aw man",

```

Comment:
...I won't insert my personal biases here... but god is this cringe

Reply:
Good point